corehq/apps/locations/util.py

```diff
@@ -209,7 +209,7 @@ def get_headers(self):
                 additional_headers.extend(self._prefix_headers('consumption', self.product_codes))
             additional_headers.append('uncategorized_data')
 
-            headers[loc_type.code] = [LOCATION_SHEET_HEADERS.keys()[:-2] + additional_headers]
+            headers[loc_type.code] = [LOCATION_SHEET_HEADERS.values()[:-2] + additional_headers]
```

Comment:
I don't think that  slice notation works in python 3 to force the iterator to a list

Reply:
This got removed later when I split `LOCATION_SHEET_HEADERS`